### PR TITLE
[LICENSE]LM

### PR DESCRIPTION
**Pull request by abby-cyber**

### CODE REVIEW

File: docs-2.0/9.about-license/2.license-management-suite/3.license-manager.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+
+```bash
+./nebula-license-manager-cli license info
+```
+
+- When the license key is not loaded, the output is as follows:
+
+  ```
+  ┌───────────────┬───────────┐
+  │ LMID          │ QPDH-xxx  │
```
Comment (veezhang) [inline docs-2.0/9.about-license/2.license-management-suite/3.license-manager.md:99]: `QPDH-xxx` 这个看看要不要对齐下？ 比如 `QPDH-XXXX` ?

File: docs-2.0/9.about-license/2.license-management-suite/3.license-manager.md [hunk 2]
```diff
@@ -0,0 +1,200 @@
+# License Manager
+
+License Manager (LM) is a service that runs on a server for you to manage licenses. You can use an LM client that communicates with the LM service to load [license keys](2.license-center.md) and view license information, including the license validity period and purchased nodes. By configuring the LM service in the NebulaGraph database and its associated software, the validity of the license can be verified to ensure the normal use of the NebulaGraph database and its associated software.
```
Comment (nianiaJR) [inline docs-2.0/9.about-license/2.license-management-suite/3.license-manager.md:3]: manage licenses => manage license. One lm only map to one license. it's a 1v1 relationship.

Comment (nianiaJR) [reply docs-2.0/9.about-license/2.license-management-suite/3.license-manager.md:3]: fix content => By configuring the LM service address in the NebulaGraph....

File: docs-2.0/9.about-license/2.license-management-suite/3.license-manager.md [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+  tar -zxvf <name.tar.gz> -C <path>
+  ```
+
+  - `<name.tar.gz>`: The name of the LM TAR package.
+  - `<path>`: The installation path for the unpacked LM. If the `-C` parameter is not specified, it defaults to the current directory.
+  
+2. In the LM installation directory, start LM.
+
+  ```bash
+  nohup ./nebula-license-manager &
```
Comment (nianiaJR) [inline docs-2.0/9.about-license/2.license-management-suite/3.license-manager.md:38]: not use nohup command here, it's not a good practice in production. Here, we just info user to use the nebula-license-manager binary to start the serivce, as for how to do it, it depends on user himself, he can use systemd or other tools.

**Review by veezhang — APPROVED** [review]

**Review by slimshadylol0303 — APPROVED** [review]

**Pull request merged**